docs/reference/component-cli_component-archive.md

```diff
@@ -1,9 +1,9 @@
-## components-cli component-archive
+## component-cli component-archive
```

Comment:
do you know why the generated name of `docs/reference/component-cli` has changed but not the other names?

Reply:
all the names has changed in `docs/reference`, see the filename above the changes, here e.g.  ...rence/component**_s_**-cli_component-archive.md → ...erence/component-cli_component-archive.md